tests/tests.js

```diff
@@ -0,0 +1,519 @@
+/* global equal, notEqual, ok, module, test */
+
+define([
```

Comment:
Did you go with require and qunit because strophe uses them?

Reply:
yes